contracts/deployment/SystemV2.sol

```diff
@@ -328,33 +335,41 @@ contract SystemV2 {
         );
         allocator.connectDeposit(address(daipsm), address(morphoDaiPCVDeposit));
 
-        // Configure PCV Oracle
+        /// Configure PCV Oracle
         address[] memory venues = new address[](4);
         venues[0] = address(morphoDaiPCVDeposit);
         venues[1] = address(morphoUsdcPCVDeposit);
         venues[2] = address(daipsm);
         venues[3] = address(usdcpsm);
+
         address[] memory oracles = new address[](4);
         oracles[0] = address(daiConstantOracle);
         oracles[1] = address(usdcConstantOracle);
         oracles[2] = address(daiConstantOracle);
         oracles[3] = address(usdcConstantOracle);
+
         bool[] memory isLiquid = new bool[](4);
         isLiquid[0] = true;
         isLiquid[1] = true;
         isLiquid[2] = true;
         isLiquid[3] = true;
+
         pcvOracle.addVenues(venues, oracles, isLiquid);
 
-        // Configure PCV Router
+        /// Configure PCV Router
         address[] memory swappers = new address[](1);
         swappers[0] = address(pcvSwapperMaker);
         pcvRouter.addPCVSwappers(swappers);
 
-        // Cleanup
+        /// Allow all addresses to execute proposals once completed
+        timelockController.grantRole(
+            timelockController.EXECUTOR_ROLE(),
+            address(0)
+        );
+        /// Cleanup
         timelockController.renounceRole(
             timelockController.TIMELOCK_ADMIN_ROLE(),
-            address(this)
+            address(this) /// TODO MAKE THIS DEPLOYER ADDRESS
```

Comment:
the contract SystemV2 itself is the deployer address, why did you mark a TODO here ?

Reply:
in the foundry deploy script I don't think that will be the case

Reply:
When we deploy to mainnet, foundry isn't going to deploy a factory contract that spins up these contracts one by one. Instead, it will deploy each contract and not deploy a factory as the factory would be way over the 24kb size limit.